content/docs/3_cookbook/0_development-deployment/0_caddy/cookbook-recipe.txt

```diff
@@ -2,7 +2,7 @@ Title: Kirby meets Caddy
 
 ----
 
-Published: 2023-10-28
+Published: 2024-04-20
```

Comment:
@lukasbestle should have a look at the Caddy config changes. I don't know enough about it. But we shouldn't change the published date here.

Reply:
Fixed it.